other/restrict-clusterrole-mutating-validating-admission-webhooks/restrict-clusterrole-mutating-validating-admission-webhooks.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: restrict-clusterrole-mutating-validating-admission-webhooks
+  annotations:
+    policies.kyverno.io/title: Restrict Clusterrole for Mutating and Validating Admission Webhooks
+    policies.kyverno.io/category: Other
+    policies.kyverno.io/severity: medium
+    policies.kyverno.io/subject: ClusterRole
+    policies.kyverno.io/description: >-
+      ClusterRoles that grant write permissions over admission webhook should be minimized to reduce powerful identities in the cluster. This policy checks to ensure write permissions are not provided to admission webhooks.
```

Comment:
Missing annotations `kyverno.io/kyverno-version` and `kyverno.io/kubernetes-version`. For more details on annotations, see [here](https://github.com/kyverno/policies/wiki/Kyverno-annotations).